src/btree/bt_ovfl.c

```diff
@@ -65,7 +65,6 @@ __wt_ovfl_read(
      */
     __wt_readlock(session, &S2BT(session)->ovfl_lock);
     if (__wt_cell_type_raw(unpack->cell) == WT_CELL_VALUE_OVFL_RM) {
-        WT_ASSERT(session, __wt_txn_visible_all(session, unpack->tw.stop_txn, unpack->tw.stop_ts));
```

Comment:
Oh -curious. Do overflow cells not have a time window?

Reply:
No; this code is the one (?) place where we ended up taking cells of both `WT_CELL_UNPACK_ADDR` and `WT_CELL_UNPACK_KV` types, and so we don't have a `WT_TIME_WINDOW`. Since it's just an assert, I removed it.